cmd/getkeystone.go

```diff
@@ -18,16 +16,15 @@ func init() {
 var getKeystoneCmd = &cobra.Command{
 	Use:   "get-keystone",
 	Short: "Get Keystone related attributes",
-	Long: `Get Keystone related attributes`,
+	Long:  `Get Keystone related attributes`,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cfgMgr := GetCfg()
 		zap.L().Debug("Get keystone password")
-		password, err := cfgMgr.GetKeystonePassword(serviceName)
+		_, err := cfgMgr.GetKeystonePassword(serviceName)
```

Comment:
@cruizen This command is meaningless in its current form. If someone runs `go-firkinize get-keystone`, they will now get an empty response. 

Let's remove the command if it isn't used, or add an exception to codeQL to allow printing the password - since that's the intended behaviour anyway.